src/pages/projectSettings/ProjectSettingsTabs.tsx

```diff
@@ -0,0 +1,116 @@
+import { useEffect } from "react";
+import styled from "@emotion/styled";
+import { H2, Disclaimer } from "@leafygreen-ui/typography";
+import { Route, useParams } from "react-router-dom";
+import { routes, ProjectSettingsTabRoutes } from "constants/routes";
+import {
+  AccessTab,
+  EventLogTab,
+  GeneralTab,
+  GitHubCommitQueueTab,
+  NotificationsTab,
+  PatchAliasesTab,
+  PeriodicBuildsTab,
+  ProjectTriggersTab,
+  VariablesTab,
+  VirtualWorkstationTab,
+} from "./projectSettingsTabs/index";
+
+export const ProjectSettingsTabs: React.FC = () => {
+  const { tab } = useParams<{ tab: string }>();
+
+  useEffect(() => {}, [tab]); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
What is this for?

Reply:
Ha, this is a holdover from `PreferencesTabs.tsx` but looks like it [should have been removed there too](https://github.com/evergreen-ci/spruce/pull/648#discussion_r573096670). I'll delete both!